### ISSUE TITLE
BCOs discrepancy

### ISSUE THREAD

**Issue by jeet-vora** [glygener/glygen-issues#1945]
@katewarner @ubhuiyan 
I am finding lots of discrepancies in the BCOs. Some of the BCOs do not have usability domain and pipeline steps for few datsets that use different scripts.
IsoglyP BCOs have incorrect usability domain. Scientific name in the usability domain is missing. The Bovine Iptmnet citation BCOs has usability domain if the main dataset.

Please review the BCOs for correctness. I am fixing ones I found correct.

eg

<img width="1149" alt="image" src="https://github.com/user-attachments/assets/85d6ce2d-866c-4226-a3d5-5a133215812a">

**Comment by ubhuiyan** [comment]
I have updated/resolved the following discrepancies: 

1. Changed _Homo sapien_ to _Homo sapiens_ in the `human_proteoform_glycosylation_sites_predicted_isoglyp.csv` dataset
2. Changed the organism common name preceding TaxID to the corresponding species name in the usability domain of MCW, UniCarbKB, and O-GlcNAc Atlas BCOs 
3. Updated Isoglyp BCO usability domain for all species to mention tool source and max EVP threshold
4. Fixed the UniCarbKB and Glyconnect Citation Dataset tags to all represent "Protein Citations" as some were not consistent. 
5. Provided a usability domain to species missing one for iPTMnet Citations datasets
6. Updated GLY_000143 fro "external" to "internal" scope.
